L1Trigger/L1THGCal/interface/be_algorithms/HGCalClusteringImpl.h

```diff
@@ -23,20 +20,20 @@ class HGCalClusteringImpl{
                       const l1t::HGCalCluster & clu, 
                       double distXY) const;
 
-    void clusterize( const edm::PtrVector<l1t::HGCalTriggerCell> & triggerCellsPtrs,
+    void clusterizeDR( const edm::PtrVector<l1t::HGCalTriggerCell> & triggerCellsPtrs,
                      l1t::HGCalClusterBxCollection & clusters 
         );
 
     /* NN-algorithms */    
     void mergeClusters( l1t::HGCalCluster & main_cluster, 
-                        const l1t::HGCalCluster & secondary_cluster ) const;
+                        l1t::HGCalCluster & secondary_cluster ) const;
```

Comment:
Why removing `const`? `secondary_cluster` is not modified in the function, so it should be `const`.